game/fx/quality_of_life.go

```diff
@@ -67,7 +67,15 @@ func Select(p *match.Player, m *match.Match, containerOwner *match.Player, conta
 }
 
 // SelectFilter prompts the user to select n cards from the specified container that matches the given filter
+//
+// Deprecated: New cards should use `fx.SelectFilterFullList`
 func SelectFilter(p *match.Player, m *match.Match, containerOwner *match.Player, containerName string, text string, min int, max int, cancellable bool, filter func(*match.Card) bool) CardCollection {
+	return SelectFilterFullList(p, m, containerOwner, containerName, text, min, max, cancellable, filter, false)
+}
+
+// SelectFilter prompts the user to select n cards from the specified container that matches the given filter.
+// It also allows to show all the other cards from the container that are unselectable
+func SelectFilterFullList(p *match.Player, m *match.Match, containerOwner *match.Player, containerName string, text string, min int, max int, cancellable bool, filter func(*match.Card) bool, showUnselectables bool) CardCollection {
```

Comment:
I think `SelectFilter` and `SelectFilterFullList` would make more sense if switched around since the `FullPart` implies that it will include everything but currently it's actually `SelectFilter` that always includes everything and `SelectFilterFullList` can optionally not include everything.

Maybe just a `ctrl + f` `fx.SelectFilter` and replace all of the old ones to `fx.SelectFilterFullList` and the cards updated here to show unselected cards can use the "new" `fx.SelectFilter`.

Or we could update `SelectFilter` to include the `showUnselectables` bool and update the cards that use it so that we only have one function for this. Might be a bit more work though.

What do you think?

Reply:
Well was hoping that eventually they are all moved in time to the new method and that can be renamed back to `SelectFilter` eventually. But I see your point on how naming is not ideal currently. Changed the old ones to `SelectFilterSelectablesOnly` as it's more appropiate. And the new ones to `SelectFilter`

Didn't go with applying the change everywhere as it's the thing I was trying to avoid in the first place.